gatsby-node.js

```diff
@@ -9,10 +9,13 @@ const appInsights = require('applicationinsights');
 const fs = require('fs');
 const siteconfig = require('./site-config');
 const matter = require('gray-matter');
+const fetch = require('node-fetch');
```

Comment:
we shouldn't need this anymore as we are using node 18+ 

Should just be able to call fetch natively!